### ISSUE TITLE
Make component widths accessible in text zoom mode

### ISSUE THREAD

**Issue by geotrev** [geotrev/undernet#142]
Some widths are hard coded in pixels. They should be converted to `em`.

1. Dropdown width 
2. Accordion max-height (generated in JS, will default to 16px for now); this needs a more comprehensive fix going forward as described in #144 
3. Modal width

Related:
4. Button font size doesn't respond to text zoom
